dojo/pipeline.py

```diff
@@ -86,6 +86,15 @@ def update_product_access(backend, uid, user=None, social=None, *args, **kwargs)
                 product, created = Product.objects.get_or_create(name=project_name, prod_type=product_type)
                 if not settings.FEATURE_AUTHORIZATION_V2:
                     product.authorized_users.add(user)
+                    for project in projects:
```

Comment:
This logic will only be called when the new authorization is not active. Since the new authorization is the default now, this logic would not be called anymore. It needs to be moved directly behind creating the product.